### ISSUE TITLE
Allow not waiting for actions

### ISSUE THREAD

**Issue by jooola** [hetznercloud/cli#833]
### TL;DR

In some use cases, waiting for actions to complete does not make sens, for example deleting a server can be done in a fire and forget manner to prevent waiting too long the resource are deleted.

We proposed users to put the process in the background to continue with their scripts, but this can lead to multiple problems:
- We do not have the information if the actions that we created, was actually created without error.
- We have inconsistent behavior when deleting resources in batches, the first 10 will have the expected behavior, but the rest of the resources will still have to wait for the first batch to complete.

- https://github.com/hetznercloud/cli/issues/613
- https://github.com/hetznercloud/cli/issues/489
- https://github.com/hetznercloud/cli/issues/284

### Expected behavior

Provide a --no-wait flag to bypass any actions waiting logic in our commands.

**Comment by c33s** [comment]
i would really like to see that. i currently changing the type of server and hcloud is doing that for nearly two hours also the gui shows that the process is initiated and at 80%. would be nice to have the possibility to just let hcloud call the api and return to shell. maybe add a new poll command to check the status.
